source/administration/changelog.md

```diff
@@ -4,6 +4,148 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v4.8
+
+Release date: 2018-03-16
+
+### Security Update  XXXXXXX
+
+- Mattermost v4.8.0 contains a security fix ranging from low to high severity. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+## Highlights
+
+### CloudFront enhanced compatibility
+
+ - Added support for configuring CloudFront to host Mattermost's static assets.
+ - Allows for improved caching performance and shorter load times for those members of your team geographically distributed throughout the world.
+
+### SAML Migration Command  ([Enterprise Edition E20](https://about.mattermost.com/pricing/))
+
+ - Added a CLI command to easily migrate users to SAML.
+
+## Improvements
+
+### Web User Interface
+ - Added a web app build hash to About Mattermost dialog to tell what version of the web app is being used.
+ - Added rendering of at-mentions by the teammate name display.
+ - Made ctrl work wherever cmd works on macOS.
+ - Switched search bar to a button in tablet view, to increase how much space is available in the channel header.
+ 
+### Performance
+ - Reduced load times by optimizing database queries and optimizing WebSocket events destined for a single user by precomputing the raw JSON for WebSocket events.
+ - Created an endpoint on iOS to upload a file as octet-stream in order to use `NSUrlSessionUploadTask` that will let us upload files larger than 20 Mb.
```

Comment:
Do we know which PR this is from? I'm not quite sure what it means..

Reply:
https://mattermost.atlassian.net/browse/MM-9556 and https://github.com/mattermost/mattermost-server/pull/8293.

Reply:
Ah, maybe we can grab the summary Justin had in the release announce?